docker/stubs/ImageMagick/ImageMagick_fuzzer.c

```diff
@@ -4,26 +4,27 @@
 #include <string.h>
 #include "MagickCore/MagickCore.h"
 
-/*
-  We use BlobToImage to load input as an image, if successful destroy image.  
-*/
+# We use BlobToImage to load input as an image, if successful destroy image.
```

Comment:
This is not valid C comment. The original `/* */` syntax is valid. `#` is used mostly by preprocessor.